src/components/innertubeClient.js

```diff
@@ -29,8 +29,8 @@ function getInnertubeEmbedPayload(videoId, playlistId, playlistIndex) {
     const payload = {
         context: {
             client: {
-                clientName: getYtcfgValue('INNERTUBE_CLIENT_NAME').replace('_EMBEDDED_PLAYER', ''),
-                clientVersion: getYtcfgValue('INNERTUBE_CLIENT_VERSION'),
+                // clientName: getYtcfgValue('INNERTUBE_CLIENT_NAME').replace('_EMBEDDED_PLAYER', ''),
```

Comment:
We are populating the object with information, but this entire object gets overridden on line 54?

Reply:
replace it with `client: getYtcfgValue('INNERTUBE_CONTEXT').client` or something like that